tst/Carnival/Providers/HookServiceProviderTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Tests\Carnival\Providers;
+
+use Carnival\Hooks\HookManager;
+use Carnival\Providers\HookServiceProvider;
+use Illuminate\Contracts\Foundation\Application;
+use Mockery;
+use Tests\Carnival\CarnivalTestCase;
+
+class HookServiceProviderTest extends CarnivalTestCase {
+
+    public function testHookServiceProviderCanRegisterHookLibraryFacade() {
+        $application = Mockery::mock(Application::class);
+        $hookServiceProvider = new HookServiceProvider($application);
+        $application->shouldReceive('bind')->with('hooks', Mockery::on(fn ($argument) => $argument() instanceof HookManager ));
+        $hookServiceProvider->register();
+        $application->shouldHaveReceived('bind')->once();
```

Comment:
I haven't figured out why `shouldReceive` assertions weren't working for me inside of Laravel, but I did get it working here in Carnival. See my PR for proof

So you should be able to update this (and any other relevant tests?) to not use `shouldHaveReceived` and move your `->once()` to the `shouldReceive` call

Reply:
Okay, I will update this.